### ISSUE TITLE
The possibility to add items and blocks through config file

### ISSUE THREAD

**Issue by meecosha** [Globox1997/EnvironmentZ#124]
### Make sure you are not opening a duplicate.

- [X] I have searched the issue tracker and did not find an issue describing my bug/feature.

### Description of your suggestion.

I want to create a modpack, which has environmentZ in it, but I want to make changes with blocks and items. I'm not sure if it's possible to add a datapack to a modpack (never found a definitive answer), and there's another mod called Thermite which allows to add the blocks through config. Is it possible to abandon the datapack system and only use configs?

Or maybe there's a way to make a modpack using the custom datapacks? Thank you for the wonderful mod!

**Comment by Globox1997** [comment]
Yes of course you can add a datapack to modpacks, that is the main work of a modpacker. Since EnvironmentZ provides a datapack system, I do not plan to add a config file solution.
